tests/slurm_plugin/test_clustermgtd.py

```diff
@@ -2112,6 +2170,794 @@ def test_update_status(self, compute_fleet_status_manager, put_item_response, ex
+            True,
+        ),
+        (
+            SlurmNode("queue1-dy-c5xlarge-1", "ip-1", "hostname", "#MIXED", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", "launch_time"),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            False,
+            {"some_node_in_replacement"},
+            "Node bootstrap error: Node is in power up state without valid backing instance",
+            True,
+        ),
+        (
+            SlurmNode("queue1-dy-c5xlarge-1", "ip-1", "hostname", "DOWN*+CLOUD+POWER", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", "launch_time"),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            True,
+            {"some_node_in_replacement"},
+            "Node bootstrap error: Resume timeout expires",
+            True,
+        ),
+        (
+            SlurmNode("queue1-dy-c5xlarge-1", "ip-1", "hostname", "DOWN#+CLOUD", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", "launch_time"),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            False,
+            {"some_node_in_replacement"},
+            None,
+            False,
+        ),
+        (
+            SlurmNode("queue1-st-c5xlarge-1", "ip-1", "hostname", "DOWN*+CLOUD", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", "launch_time"),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            False,
+            {"some_node_in_replacement"},
+            None,
+            False,
+        ),
+        (
+            SlurmNode("queue1-st-c5xlarge-1", "ip-1", "hostname", "DOWN*+CLOUD", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", datetime(2020, 1, 1, 0, 0, 0)),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            False,
+            {"some_node_in_replacement"},
+            None,
+            False,
+        ),
+        (
+            SlurmNode("queue1-dy-c5xlarge-1", "ip-1", "hostname", "DOWN+CLOUD+POWER", "queue1"),
+            {
+                "ip-1": EC2Instance("id-1", "ip-1", "hostname", "launch_time"),
+                "ip-2": EC2Instance("id-2", "ip-2", "hostname", "launch_time"),
+            },
+            False,
+            {"queue1-st-c5xlarge-1"},
+            None,
+            False,
+        ),
+    ],
+    ids=[
+        "static_self_terminate",
+        "static_timeout",
+        "dynamic_self_terminate",
+        "dynamic_timeout",
+        "dynamic_runinstance",
+        "static_runinstance",
+        "static_joined_cluster",
+        "dynamic_reset_incorrect",
+    ],
```

Comment:
Can we try to test some "normal" down nodes and make sure they are not considered as bootstrap failure nodes? For example, dynamic nodes in `DRAIN+CLOUD`or `DOWN+CLOUD+POWERING_DOWN`